src/icons/components/index.ts

```diff
@@ -161,23 +161,24 @@ export {default as RemotePublication} from './RemotePublication';
 export {default as Replay} from './Replay';
 export {default as Reply} from './Reply';
 export {default as Report} from './Report';
+export {default as ResponsiveDevices} from './ResponsiveDevices';
 export {default as RichText} from './RichText';
 export {default as Rocket} from './Rocket';
 export {default as RotateLeft} from './RotateLeft';
 export {default as RotateRight} from './RotateRight';
-export {default as Save} from './Save';
-export {default as School} from './School';
 export {default as SdLgenerator} from './SdLgenerator';
 export {default as SdLreport} from './SdLreport';
+export {default as Save} from './Save';
```

Comment:
Looks like alphabetical sorting got reshuffled here (same for the other changes in this file); could be good to reorder back.

Reply:
SVGR automatically generates the index file, I don't know why it doesn't respect the alphabetical order